charts/ratify/templates/configmap.yaml

```diff
@@ -15,9 +15,5 @@ data:
         "verificationRequestTimeout": {{ .Values.provider.timeout.validationTimeoutSeconds | int | mul 1000 | add -100 }},
         "mutationRequestTimeout": {{ .Values.provider.timeout.mutationTimeoutSeconds | int | mul 1000 | add -50 }}
       },
-      "crl": {
-        "cache": {
-          "enabled": {{ default "true" .Values.crl.cache.enabled }}
```

Comment:
I feel the root cause is helm will regard `false` bool value as not set in default function. One workaround is to let user pass in "false" string instead of bool value. But it's inconsistent with other bool vars. Maybe we don't need the default function here, since we already set the default value to true in values.yaml.
"enabled": {{ .Values.crl.cache.enabled }} might be enough.
@susanshi

Reply:
fixed